BLSQ/iaso
# POLIO-1525 POLIO-1526: Target population

Target population filed should move from risk assessment  tab to round tab like in the mock-up
Also Calendar popper should display target population , description  + remove budget and vaccine 

Related JIRA tickets : POLIO-1525 POLIO-1526

## Self proofreading checklist


## Changes

- change calendar serializers to return target population per round and campaign description
- move target population filed to round tab
- review calendar popup to add/remove fields

## How to test

- Edit a campaign description and change the target population for a round
- Open calendar and click on the round you just changed
- you should see target population and description on the popup

## Print screen / video

<img width="1504" alt="Screenshot 2024-03-26 at 13 20 01" src="https://github.com/BLSQ/iaso/assets/12494624/d8da16da-23f4-4fc7-8d89-ed576b11e7a7">
<img width="1658" alt="Screenshot 2024-03-26 at 13 16 35" src="https://github.com/BLSQ/iaso/assets/12494624/eb512eb8-0704-4a0d-b363-f481ac396998">
